package_io/input_parser.star

```diff
@@ -96,6 +96,12 @@ def parse_input(input_args):
 	if result["network_params"]["deneb_fork_epoch"] == 0:
 		fail("deneb_fork_epoch is 0 needs to be > 0 ")
 
+	if result["network_params"]["electra_fork_epoch"] != None and result["network_params"]["deneb_fork_epoch"] < result["network_params"]["electra_fork_epoch"]:
```

Comment:
can deneb and electra be the same say 5,5 - or should we fail that as well?

Reply:
So after some discussion we agreed that deneb can't happen when electra is defined. Adjusted accordingly